commands/commands_test.go

```diff
@@ -196,15 +196,15 @@ func withTestClient(t *testing.T, tFn testFn) {
 }
 
 type TestConfig struct {
-	SSHFn func(user, host, keyPath string, port int) runner.Runner
+	SSHFn func(user, host, keyPath string, port int, agentForwarding bool) runner.Runner
```

Comment:
I'm thinking this won't be the last option for SSH, so shouldn't we pass this option in a more forward-thinking way?

```go
type SSHOptions map[string]interface{}

func(user, host, keyPath string, port int, options SSHOptions) runner.Runner {}
```

This would allow you do something like:

```go
opts[doctl.ArgsSSHAgentForwarding] = true
```

Not sure what this means long term, but it would solve the immediate issue of longer function signatures.